src/utils/utils.ts

```diff
@@ -0,0 +1,20 @@
+import {ListItemColumn, UserData} from 'types/types';
+
+export const getUserColumns = (user: UserData): ListItemColumn[] => {
```

Comment:
created this method because it is used in more than one component, so it can be exported here to be used anywhere it needs to be used